examples/speed_tester/speed_tester.c

```diff
@@ -400,6 +401,8 @@ nf_setup(struct onvm_nf_info *nf_info) {
                 }
 
                 packet_number = (packet_number ? packet_number : MAX_PKT_NUM);
```

Comment:
can you also change this to 
`packet_number = (use_custom_pkt_count ? packet_number : MAX_PKT_NUM);`